packages/react-scripts/template/src/app.js

```diff
@@ -0,0 +1,27 @@
+// import React, { Component } from 'react'
```

Comment:
you probably don't want to keep these commented lines

Reply:
I keep them so this part would be either to rewrite:
 - `Show main screen with logo that rotate`